src/AppController.m

```diff
@@ -586,6 +586,25 @@ - (void)updaterWillRelaunchApplication:(SUUpdater *)updater
 	
 }
 
+/* feedURLStringForUpdater:
+ * This is a delegate for Sparkle.framework
```

Comment:
This should go in a separate class - something like `ViennaSparkleDelegate.m`. AppController is our most bloated class and we need to trim it down.